src/v0/destinations/am/transform.js

```diff
@@ -297,6 +297,15 @@ const identifyBuilder = (message, destination, rawPayload) => {
       }
     });
   }
+  // update identify call request with unset fields
+  const fieldsToUnset = get(message, 'integrations.Amplitude.fieldsToUnset');
```

Comment:
can we move whole thing to a separate function

Reply:
so that we can write unit tests

Reply:
Ok

Reply:
done